sources/kube.go

```diff
@@ -259,6 +272,7 @@ func newKubeSource() (*KubeSource, error) {
 		client:      kubeClient,
 		lastQuery:   time.Now(),
 		kubeletPort: *argKubeletPort,
+		activePods:  make(map[PodInstance]bool),
```

Comment:
yes, this is a problem because we don't keep a list of pods being tracked. Would it be good enough to regenerate the list in getPods() - purge all old entries that are not present in the new listing from getPods()?